### PR TITLE
Added Zulip Project

### PR DESCRIPTION
**Pull request by Akash1362000**

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -696,6 +696,13 @@ const projectList = [
     projectLink: 'https://github.com/muke1908/chat-e2ee/labels/good%20first%20issue',
     description: 'Chat app in end-to-end enctypted environment without registration',
     tags: ['React', 'Chat', 'Javascript', 'Beginner', 'NodeJS'],
+  },
+  {
+    name: 'Zulip',
+    imageSrc: 'https://avatars0.githubusercontent.com/u/4921959?s=200&v=4',
+    projectLink: 'https://github.com/zulip',
```
Comment (Roshanjossey) [inline src/components/ProjectList/listOfProjects.js:703]:
```suggestion
    projectLink: 'https://github.com/zulip/zulip/contribute',
```

**Review by Roshanjossey — CHANGES_REQUESTED** [review]
Good job. Zulip was in my mind for some time now.

I think it's better to show users the issues than to take them to organisation page. Please make the suggested change

**Comment by Akash1362000** [comment]
Made the requested changes.

**Comment by Roshanjossey** [comment]
Awesome. Thank you

**Pull request merged**